### DISCUSSION TITLE
Unable to get the data using ss.PV.get()

### DISCUSSION THREAD

**Discussion by JunejaAmmy** [CURENT/andes#256] [Q&A] [answered]
Hi Dr. Cui,

I was trying to fetch the data of PV sheet using following command:
`ss.PV.get("p0", "2", attr='vin')`

Here, header or column name is **p0**,
idx is **2** and
attr is **vin**

and I'm getting the error: 'PV: device not exist with idx=2.' But the idx = 2 exists as you can see below.

![image](https://user-images.githubusercontent.com/97614806/151416777-dc1751cc-bbd6-407c-9f25-7fcb8b0cda4d.png)

Thanks

**Comment by cuihantao** [comment, marked as answer]
You can check `ss.PV.idx.v` and tell the values are integers, so `ss.PV.get("p0", 2, attr='vin')` should do.

**Reply by cuihantao** [reply]
Also see: https://docs.andes.app/en/stable/tutorial.html#data-consistency

**Reply by JunejaAmmy** [reply]
Thanks Dr. Cui
